camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/CatalogYamlEntityTest.java

```diff
@@ -50,15 +48,6 @@
 
 
 public class CatalogYamlEntityTest extends AbstractYamlTest {
-    
-    private static final String SIMPLE_ENTITY_TYPE = OsgiTestResources.BROOKLYN_TEST_OSGI_ENTITIES_SIMPLE_ENTITY;
-    private static final String MORE_ENTITIES_POM_PROPERTIES_PATH =
-        "META-INF/maven/org.apache.brooklyn.test.resources.osgi/brooklyn-test-osgi-more-entities/pom.properties";
-
-    @Override
-    protected boolean disableOsgi() {
-        return false;
```

Comment:
@geomacy This appears to be unnecessary, and it slows down testing dramatically, and causes a lot more GC messages. It might be the reason for the testing failures in #338 (possibly compounded by not de-duping the search paths, or possibly not).

cc @aledsage @neykov As we do more with OSGi we should be aware that starting tests that use OSGi each take almost 1s.  Also we're using the "fake" OSGi I think (felix not karaf); maybe it makes sense for some modules to launch in OSGi when they run their tests, and/or maybe there are some other optimizations?